core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -189,6 +191,67 @@ oppia.controller('CreatorDashboard', [
       return UrlInterpolationService.getStaticImageUrl(iconUrl);
     };
 
+    $scope.showCreateQuestionModal = function() {
+      var question = QuestionObjectFactory.createDefaultQuestion();
+      TopicsAndSkillsDashboardBackendApiService.fetchDashboardData().then(
+        function(response) {
+          var topicSummaries = response.data.topic_summary_dicts;
+          $uibModal.open({
+            templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+              '/pages/creator_dashboard/create_question_modal_directive.html'),
+            backdrop: true,
+            size: 'lg',
+            resolve: {},
+            controller: [
+              '$scope', '$uibModalInstance', function(
+                  $scope, $uibModalInstance) {
+                $scope.question = question;
+                $scope.topicId = null;
+                $scope.questionStateData = $scope.question.getStateData();
+                $scope.topicSummaries = topicSummaries;
+
+                $scope.isNotValidQuestion = function() {
+                  return !$scope.question.validate();
+                };
+
+                $scope.dismissModal = function() {
+                  $uibModalInstance.dismiss();
+                };
+
+                $scope.createQuestion = function() {
+                  $uibModalInstance.close({
```

Comment:
Perhaps you should validate here that topicId is not null and that the question is valid.

Reply:
Done